spec/features/process_a_request_reject_spec.rb

```diff
@@ -13,6 +13,19 @@
     travel_to(Date.new(2018, 4, 5)) { ex.run }
   end
 
+  describe 'when the prisoner is not registered at the prison', vcr: { cassette_name: 'prisoner_not_at_given_prison', record: :new_episodes } do
```

Comment:
Is 'record_new_episodes' necessary here?

Reply:
nope 😅